ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/BookingsApiClientHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using RestSharp;
+using ServiceWebsite.AcceptanceTests.Configuration;
+using ServiceWebsite.AcceptanceTests.Models;
+using ServiceWebsite.AcceptanceTests.TestClients;
+using ServiceWebsite.BookingsAPI.Client;
+using ServiceWebsite.Configuration;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class BookingsApiClientHelper
+    {
+        public BookingsApiTestClient CreateClientWithConfig(IConfigurationRoot configRoot)
+        {
+            var vhServiceConfig = Options.Create(configRoot.GetSection("VhServices").Get<ServiceSettings>()).Value;
+            var token = ConfigurationHelper.GetBearerToken(configRoot);
+            var client = new RestClient(vhServiceConfig.BookingsApiUrl);
+            return new BookingsApiTestClient(client, token);
```

Comment:
I dont think you need to return this out, if you initialise this in a constructor, then the methods in this class can use the field and not need it passed in to the mnethod

Reply:
done